puppet/modules/web/manifests/vhost/deb.pp

```diff
@@ -35,5 +36,8 @@
       mode    => '0700',
       content => template('freight/deploy_debs.erb'),
     }
+
+    User <| title == 'freightstage' |> -> User <| title == $user |>
+    User <| title == $user |> { groups +> ['freightstage'] }
```

Comment:
Rather than setting it this way, should `secure_ssh::receiver_setup` gain a parameter to set the groups on the user?